### DISCUSSION TITLE
Deta Space Features

### DISCUSSION THREAD

**Discussion by s4m-mo** [deta/space-docs#24] [Ideas]
Would these be possible?

 - Ability to delete collections
 - Ability to clear Micro logs
 - Ability to view Micro logs fullscreen (they are difficult to read on the small, narrow, scrolled panel)

Thanks for your time!

**Comment by abdelhai** [comment]
- yes soon
- good idea. we need to think but likely at some point
- agree, would be great. will take note

cc @BetaHuhn @mikBighne98

**Reply by s4m-mo** [reply]
Thanks so much
